.github/workflows/codegen.yml

```diff
@@ -8,18 +8,80 @@ name: codegen
 jobs:
   discovery:
     uses: googleapis/discovery-artifact-manager/.github/workflows/list-services.yml@master
-
-  generate:
+  batch:
     runs-on: ubuntu-20.04
     needs: discovery
+    outputs:
+      services: ${{ steps.chunk.outputs.result }}
+    steps:
+      - uses: actions/github-script@v5
+        id: chunk
+        with:
+          script: |
+            console.log('splitting service names list into batches')
+            const services = ${{ needs.discovery.outputs.services }}
+            const result = {
+              "one": [],
+              "two": [],
+            };
+            for (let i = 0; i < services.length; i++) {
+              if (services[i] != "discovery" && services[i] != "websecurityscanner") {
```

Comment:
consider abstracting this out to an array say `excludedServices`

Reply:
```suggestion
              if (!["discovery", "websecurityscanner"].includes(services[i])) {
```